src/components/formik/FormikComponent.tsx

```diff
@@ -16,6 +16,7 @@ interface Props {
 
 function FormikComponent(props: Props) {
   const { formSections, initialValues, schema } = props;
+  let history = useHistory();
```

Comment:
We prefer using `const` to `let`. I'll be setting up eslint soon so hopefully that will be more clear.
```suggestion
  const history = useHistory();
```